src/app/controllers/userActiveGames.controller.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * This is the controller file for the puzzle endpoint
+ * This file is called by the router file and calls the service file
+ * There are four main functions {@link createPuzzle}, {@link searchPuzzle},
+ * {@link updatePuzzle}, and {@link removeUserActiveGame}
+ * The main purpose of the controller is to make sure that only validated and sanitized data
+ * moves on to the service function
+ * @module UserActiveGamesController
+ */
+
+import {matchedData} from "express-validator";
+const userActiveGamesService = require('../services/userActiveGames.service');
+
+/**
+ * Returns 201 if userActiveGameService is successful
+ * Otherwise catches error and sends to our errorHandler
+ * Takes sanitized input and sends it to userActiveGamesService
+ * @param req This is the request object
+ * @param res This is the response object
+ * @param next This takes us to the errorHandler if request fails
+ */
+async function createUserActiveGame(req, res, next) {
+
+    const allData = Object.values(matchedData(req, { locations: ['body'] }));
+    console.log(allData.values());
+    // this is needed because the last element in matchedData array is the original request for some reason.
+    // I believe this is a bug with express-validator relating to bodies that have arrays
+    console.log(allData);
+    console.log("Hello!" + allData);
```

Comment:
allData is printed twice

Reply:
Thanks for catching that, that is old debug code.